### ISSUE TITLE
Escape pods will not launch automatically

### ISSUE THREAD

**Issue by ApolloDensin** [PolarisSS13/Polaris#3898]
#### Brief description of the issue
Call emergency shuttle, get in escape pods large or small, will not launch automatically when the escape shuttle launches
#### What you expected to happen
I expect the pods to launch with the shuttle
#### What actually happened
They stay in dock, and need to manually be launched
#### Steps to reproduce
call escape shuttle, get in pod, wait for timer to finish. shuttle does not launch
#### Additional info:
 - **Server Revision**: Found using the "Show Server Revision" verb under the OOC tab.
 - **Anything else you may wish to add** (Location if it's a mapping issue, etc)

**Comment by SilverTalismen** [comment]
This was due to the escape or transfer shuttle being stuck because the airlocks need to safely close and pressurize which they will be unable to do is one or more is destroyed, damaged or unpowered.

**Comment by ApolloDensin** [comment]
is this supposed to prevent all of them from launching then? because it seems if one is messed up they are all messed up

**Comment by SilverTalismen** [comment]
Yeah all of the pods are tied to the escape/transfer shuttle leaving, if the shuttle can't leave then the pods won't.

**Comment by TheFurryFeline** [comment]
On a test as I've reported in #3455, I can confirm that pods launch automatically now. Was in the cryo storage room after admin-calling evac. All pods seemed to be at the evac centre, as well.
